app/data/models/customer.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize, DataTypes) => {
+  const customer = sequelize.define('customer', {
+    customerId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    traderId: DataTypes.STRING,
```

Comment:
This might be a bit pedantic.  But maybe call this `trader`.  Just as that's what it's referred to in messages and other databases storing the same data.